### PR TITLE
Update nodejs version to 12 - Closes #429

### LINKED ISSUE #429: Update Node JS to latest LTS
We updated Node to version 12 via LiskHQ/lisk-sdk#4448 we need to update the documentation to reflect this change.

### PR DESCRIPTION
**Pull request by chris529**
Closes #429

### CODE REVIEW

File: modules/ROOT/partials/install-node.adoc [hunk 1]
```diff
@@ -23,11 +24,11 @@ It is recommended to use a Node version manager, such as https://github.com/crea
 NVM is a bash script which enables the management of multiple active Node.js versions.
 
 . To install nvm, please follow these https://github.com/creationix/nvm#install--update-script[instructions].
-. To install the correct version of Node.js using NVM, execute the command listed below:
+. Please install the latest version of Node.js using NVM, and execute the command listed below:
```
Comment (Tschakki) [inline modules/ROOT/partials/install-node.adoc]:
```suggestion
. Install the latest LTS version of Node.js:
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
In the meantime, there was a new release of nodeJS, let's refer to the newest v12 LTS: `12.15.0` 

Also, in in transport tutorial setup, we still recommend nodejs v10.

Please update this again, thanks!

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ Lisk Elements is a JavaScript library containing several separately installable
 
 === Supported Platforms
 
-* Platforms with Node.js version 8.10 and above.
+* Platforms with Node.js version 12.14.1 and above.
```
Comment (chris529) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]:
```suggestion
* Platforms with Node.js version 12.15.0 and above.
```

File: modules/ROOT/pages/tutorials/cashback.adoc [hunk 3]
```diff
@@ -60,7 +60,7 @@ The next step is to install the `lisk-sdk` package and add it to the project's d
 |===
 |Dependencies |Version
 
-|Node.js |10.17.0
+|Node.js |12.14.1 or higher
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/cashback.adoc]:
```suggestion
|Node.js |12.15.0 or higher
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ IMPORTANT: If some of the dependencies are missing, please go to xref:{url_setup
 [options="header",]
 |===
 |Dependencies |Version
-|Node.js |10.17.0
+|Node.js |12.14.1 or higher
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
|Node.js |12.15.0 or higher
```

File: modules/ROOT/partials/install-node.adoc [hunk 5]
```diff
@@ -23,11 +24,11 @@ It is recommended to use a Node version manager, such as https://github.com/crea
...
 
 [source,bash]
 ----
-nvm install 10.17.0
+nvm install 12.14.1
```
Comment (chris529) [inline modules/ROOT/partials/install-node.adoc]:
```suggestion
nvm install 12.15.0
```

File: modules/ROOT/partials/install-node.adoc [hunk 6]
```diff
@@ -47,7 +48,7 @@ sudo apt-get install -y nodejs
 
 [source,bash]
 ----
-brew install node@10.17.0
+brew install node@12.14.1
```
Comment (chris529) [inline modules/ROOT/partials/install-node.adoc]:
```suggestion
brew install node@12.15.0
```

**Review by chris529 — COMMENTED** [review]
Changed to 12.15.0

**Review by Tschakki — CHANGES_REQUESTED** [review]
One small thing, in transport tutorial, we still advice to install node v10

**Review by Tschakki — APPROVED** [review]

**Pull request merged**